### DISCUSSION TITLE
Add some keywords (or something else, not sure how it's coded) to allow Dynamic Surroundings mod recognize material type for step sounds

### DISCUSSION THREAD

**Discussion by xenoramos** [Roadhog360/Et-Futurum-Requiem#270] [Improvements]
Hello. Just curious, is it possible to add something (I assume some "keywords", but unsure) into block properties so DS would recognize the type of material for steps? Some blocks already have it set correctly (like Cobbled Deepslate with stone-floor step sounds, or Chorus Fruit with wooden-type sound) and are working nicely, but other blocks (e.g. other types of Deeplate) are missing sounds at all.
Of course if it can be done easily without rewriting anything~

**Comment by Roadhog360** [comment]
I do not know how to do this. It's not really my fault dynamic surroundings has a very bad system for how other mods are given sounds.

Unless somebody can tell me how to do this in great detail, this is not planned.

P.S.: I also have low motivation to actually do this myself because I really don't like the alternate footstep sounds that it introduces; I think they sound unfitting.

**Reply by xenoramos** [reply]
Alright, understood, thanks for the reply :)

**Comment by GregoriusT** [comment]
Try disabling the config for new Block Sounds, that might help.

But I am pretty sure that Mod might only work for vanilla Step Sounds for some dumb reason, completely ignoring custom modded ones.

**Reply by xenoramos** [reply]
I would prefer having these step sounds, and I completely lacked literally any step sounds besides DS mod's. Roadhog360's message made me realize that it is actually a problem in my settings, and I should have sounds even without DS (I've played with DS for several years now, so got used to it so much). So I disabled DS's step sounds config and started looking. And found out that for some reason, step sounds are assigned to "friendly creatures" config on Minecraft sound settings, and I had these turned off (because of some farms with many friendly mobs), that's why I struggled to make non-DS step sound work. I somewhat changed my settings, so I now get non-DS step sounds working

**Reply by Roadhog360** [reply]
"Friendly creatures" is labeled "neutral" internally. All block walk sounds use this category, even the vanilla ones.
